.azure-pipelines.yml

```diff
@@ -17,7 +17,7 @@ variables:
   MAX_TV_DEVICE_NAME: Apple TV 4K
   MAX_IPAD_DEVICE_NAME: iPad Pro (11-inch) (2nd generation)
   DEFAULT_NODE_VERSION: "10.x"
-  DEFAULT_RUBY_VERSION: ">= 2.6"
+  DEFAULT_RUBY_VERSION: "2.7"
```

Comment:
seems line unit tests are not very happy

Reply:
Unit tests failed by the below lines. Will take a look.
(Previously Azure tried to install dependencies with Ruby 3.0, but fastlane should have not worked/succeeded to install with it)
```
  /Users/runner/work/1/s/WebDriverAgentLib/Vendor/CocoaAsyncSocket/GCDAsyncSocket.m:707:13: variable length array used [-Wvla]

        uint8_t seq[termLength];
^



⚠️  /Users/runner/work/1/s/WebDriverAgentLib/Vendor/CocoaAsyncSocket/GCDAsyncSocket.m:7139:25: variable length array used [-Wvla]

                SSLCipherSuite ciphers[numberCiphers];
```